src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml.cs

```diff
@@ -97,10 +97,17 @@ private void ExecuteSearchElement(ListBoxItem listBoxItem)
                 if (searchElement.CreateAndConnectCommand.CanExecute(port.PortModel))
                 {
                     searchElement.CreateAndConnectCommand.Execute(port.PortModel);
+                    var nodeName = (searchElement.Model is Search.SearchElements.NodeModelSearchElement) ? searchElement.FullName : searchElement.Model.CreationName;
```

Comment:
If it is a nodemodel node, I think it needs to be in this format https://github.com/DynamoDS/Dynamo/blob/bf29f561dba76d285b5478ae70173f0c6d283eec/src/DynamoCore/Search/SearchElements/MLNodeSearchElement.cs#L255 like how it is stored in dyn file. FullName and AssemblyName

Reply:
Updated